secret-squirrel.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
 	files: {
 		allow: [
 			'dotfiles/.stylelintrc',
+			'dotfiles/.editorconfig',
```

Comment:
🤔 weird this wasn't needed before

Reply:
Yeah I'm not sure why this started blocking me?